walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,125 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.*;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        //the time slots that mandatory attendees are unavailable
```

Comment:
nit: usually comments are of this form:
// Start with a capital, end with a period, space between the slashes and the text.

I don't care too much: I think consistency is the most important thing. But for the capstone I would like to see something closer to my suggestion used throughout.